docs/apis/streaming/windows.md

```diff
@@ -24,1023 +24,593 @@ specific language governing permissions and limitations
 under the License.
 -->
 
+Flink uses a concept called *windows* to divide a (potentially) infinite `DataStream` into slices
+based on the timestamps of elements or other criteria. This division is required when working
+with infinite streams of data and performing transformations that aggregate elements.
+
+<span class="label label-info">Info</span> We will mostly talk about *keyed windowing* here, this
```

Comment:
1) Explain the "keyed" a bit, i.e. that we assume that the input streams are of the form <K,V>.
2) Full stop after the here.